govc/namespace/create.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright (c) 2024-2024 VMware, Inc. All Rights Reserved.
```

Comment:
just 2024, not 2024-2024?

Reply:
This bothers me as well but it is enforced by `reviewdog`

```  reviewdog: Reporting results for "Go headers"
  reviewdog: found at least one result in diff
  Error: [Go headers] reported by reviewdog 🐶
  found mismatched boilerplate lines:
  
  Raw Output:
  govc/namespace/create.go:2: found mismatched boilerplate lines:
  {[]string}[0]:
  	-: "Copyright (c) 2024-2024 VMware, Inc. All Rights Reserved."
  	+: "Copyright (c) 2024 VMware, Inc. All Rights Reserved."
  Error: Process completed with exit code 1.
```

The configuration for this is in `boilerplate.go.txt` and it requires each new file to have a start and an end copyright year.
It looks like that this rule is only being run on new files and since it was added in 2022 we have lots of old files that use a "single-year" header

Reply:
Should we update `boilerplate.go.txt` to allow for both?

Reply:
I think this decision should be made by the community and addressed separately from this change.
@dougm what do you think?

Reply:
Yes, let's address separately. I'm ok with it as-is, only notice it with new files in a PR. If someone who knows the GH Actions / reviewdog magic to change it, that's cool too.